UI/src/components/templates/capone.js

```diff
@@ -9,8 +9,8 @@
         .module(HygieiaConfig.module)
         .controller('CapOneTemplateController', CapOneTemplateController);
 
-    CapOneTemplateController.$inject = [];
-    function CapOneTemplateController() {
+    CapOneTemplateController.$inject = ['$interval'];
```

Comment:
@kheinz04 do we need to inject $interval here

Reply:
Good catch, that was from another update when I was trying to loop the performance tab as well. I will remove.